dags/pytorch_xla/configs/pytorchxla_torchbench_config.py

```diff
@@ -242,6 +273,17 @@ def get_nvidia_driver_install_cmd(driver_version: str) -> str:
     )
     return nvidia_driver_install
 
+  install_torch_xla2_dependency = (
+      (
+          # TODO(piz): torch_xla2 only support nightly test at this time.
+          "pip install torch_xla[pallas] -f https://storage.googleapis.com/jax-releases/jax_nightly_releases.html -f https://storage.googleapis.com/jax-releases/jaxlib_nightly_releases.html",
```

Comment:
What's the reason for installing `torch_xla[pallas]`? `torch_xla2` doesn't depend on the main package.

Reply:
Just want to use the jax, libtpu version from what is specified in torch_xla uniformly. This helps us use the same script to test on both torch_xla and torch_xla2. WDYT?

Reply:
Let's let them specify JAX versions separately. Even if it works now, it will break in the future. To use the same VM with two different sets of packages, I recommend creating a venv: https://docs.python.org/3/library/venv.html

Reply:
Thanks, I will add creating the env for different test as a TODO. Currenly I have isolated the torch_xla and torch_xla2 benchmarking dependencies in https://github.com/pytorch/xla/pull/7144. We don't install torch_xla[pallas] in the script anymore.

However, one thing I want to mention is that using the same libtpu version between torch_xla and torch_xla2 can give us a fail comparsion between those two.

Reply:
Based on offline discussion, we uninstaill the libtpu from torch_xla and reinstall based on torch_xla2. We don't care about the libtpu version difference in this case.